set.mm

```diff
@@ -751469,4 +751469,97 @@ with simplest antecedents (i.e., in the corresponding ordering of binary
 
 $( (End of Adhemar's mathbox.) $)
 
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Mathbox for Igor Ieskov
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+  ${
+    binom2d.1 $e |- ( ph -> A e. CC ) $.
+    binom2d.2 $e |- ( ph -> B e. CC ) $.
+    $( Deduction form of binom2.  (Contributed by Igor Ieskov, 14-Dec-2023.) $)
+    binom2d $p |- ( ph -> ( ( A + B ) ^ 2 ) =
+                ( ( ( A ^ 2 ) + ( 2 x. ( A x. B ) ) ) + ( B ^ 2 ) ) ) $=
+      ( cc wcel caddc co c2 cexp cmul wceq binom2 syl2anc ) ABFZGCPGBCHZIJZKZIB
+      RSIRBCLZITIQICRSIQIMDEBCNO $.
+  $}
+
+  ${
+    abc3i.1 $e |- A e. CC $.
+    abc3i.2 $e |- B e. CC $.
+    abc3i.3 $e |- C e. CC $.
+    $( Cube of sum of three numbers.  (Contributed by Igor Ieskov,
+       14-Dec-2023.) $)
+    abc3i $p |- ( ( ( A + B ) + C ) ^ 3 ) =
```

Comment:
I propose to call this theorem cu3addi ("cu" fore "cube" as in ~cu2, "3add" for sum of three numbers" as in ~lt3addmuld)

Reply:
Renamed. Thank you.